src/Arcus.Templates.Tests.Integration/WebApi/Health/v1/TimeSpanConverter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Arcus.Templates.Tests.Integration.WebApi.Health.v1 
+{
+    /// <summary>
+    /// JSON converter to parse <see cref="TimeSpan"/> values.
+    /// Fixing issue: https://github.com/JamesNK/Newtonsoft.Json/issues/2189.
+    /// </summary>
+    public class TimeSpanConverter : JsonConverter<TimeSpan>
```

Comment:
Is there no built-in way? That would be strange.

Reply:
See issue. Apperently a problem since .NET Core 3.0.

Reply:
Looked over that one, sorry.